### PR TITLE
Icon widget.

### PR DESCRIPTION
**Pull request by braamgenis**
To resolve GH-154.

### CODE REVIEW

File: widgets/icon/icon.php [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+			),
+		);
+	}
+
+	function get_style_name( $instance ) {
+		return 'sow-icon';
+	}
+
+	function get_less_variables( $instance ) {
+		$les = ';asda';
```
Comment (gregpriday) [inline widgets/icon/icon.php]: What's this do?

File: widgets/icon/tpl/icon.php [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * @var $icon
```
Comment (gregpriday) [inline widgets/icon/tpl/icon.php]: Aaah nice, I need to start doing this.

File: widgets/icon/styles/sow-icon.less [hunk 3]
```diff
@@ -0,0 +1,14 @@
+@color: h1;
```
Comment (gregpriday) [inline widgets/icon/styles/sow-icon.less]: Ideally, these defaults should make sense. So the actual default color as `@color`. Same for `@size`. Can't remember if this has any benefit, but it just feels neater.

File: widgets/icon/tpl/icon.php [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+<div id="sow-icon-container" class="sow-icon">
+	<?php if ( ! empty( $url ) ) { ?>
+	<a href="<?php esc_attr_e( $url ) ?>" <?php if ( ! empty( $new_window ) ) echo 'target="_blank"'; ?>>
+	<?php } ?>
+		<?php
+		echo siteorigin_widget_get_icon( $icon );
+		?>
+	<?php if ( ! empty( $url ) ) { ?>
+	</a>
+<?php } ?>
```
Comment (gregpriday) [inline widgets/icon/tpl/icon.php]: One level of indenting on this line would be a little neater.

File: widgets/icon/icon.php (hunk 1 above)
Comment (braamgenis) [inline widgets/icon/icon.php]: Let's me put breakpoints in the `get_less_variables` function. I guess I should clean up after myself. :sweat_smile:

**Pull request merged**